scss/civicrm/campaign/_campaign.scss

```diff
@@ -10,3 +10,5 @@
     }
   }
 }
+
+@import "pages/survey";
```

Comment:
this can't be here, we should have `@import`s only in the main file

Reply:
We include sub parts of the module like this
i.e. `search/pages` are included via `search/_search.scss`
This helps us to have clear view on how sub-parts are managed.
see: https://github.com/civicrm/org.civicrm.shoreditch/blob/develop/scss/civicrm/search/_search.scss